### ISSUE TITLE
not reading tags correctly?

### ISSUE THREAD

**Issue by frnsys** [arybczak/ncmpcpp#146]
I'm using mopidy and I'm having an issue where title tags show up as `<empty>` in ncmpcpp and the album tags are always shown as the artist name (here the album should be "Power Supply"):

![](https://discuss.mopidy.com/uploads/default/optimized/1X/7da801d1d71e8f8589e7e2f27f9ace2c56856f84_1_690x252.png)

I thought it might be an issue with mopidy but it appears to be sending the correct data, e.g.:

![2016-09-13-103000_654x285_scrot](https://cloud.githubusercontent.com/assets/1059947/18477839/129ec5b2-799d-11e6-81b9-7406404b3c19.png)

I've tried with ncmpcpp 0.8_dev, 0.7.4, and 0.6.6.

**Comment by frnsys** [comment]
this seems to be related, but for some reason the Album Artist tag is being displayed as the Album tag.

This is the data from mopidy:
![2016-09-13-115057_632x286_scrot](https://cloud.githubusercontent.com/assets/1059947/18480931/7264d10c-79a8-11e6-8f8a-5d7d199298f5.png)

here's what shows up in ncmpcpp:
![2016-09-13-115117_673x101_scrot](https://cloud.githubusercontent.com/assets/1059947/18480943/7c8a0cba-79a8-11e6-80fc-62e2be583f8a.png)

**Comment by frnsys** [comment]
I've just tried `ncmpc` and that seems to handle the tags from mopidy fine.

**Comment by arybczak** [comment]
Did you set custom song_columns_list_format?

**Comment by arybczak** [comment]
Can you check how these files look like in song info screen?

**Comment by frnsys** [comment]
sorry, I forgot to close this - IIRC, it was because mopidy is using an old version of the mpd protocol...I am on ncmpcpp 0.5.10 and it works fine now.

**Comment by arybczak** [comment]
This is weird, I don't see the reason it works in 0.5.10 and doesn't in newer ones.

**Comment by frnsys** [comment]
I just revisited this and figured out what was causing the problem...I was using the latest version of `libmpdclient`, installed this way:

```
git clone --depth=1 git://git.musicpd.org/master/libmpdclient.git /tmp/libmpdclient
cd /tmp/libmpdclient
./autogen.sh
make
sudo make install
```

then building `ncmpcpp` (0.7.7) with that.

I uninstalled that and installed `libmpdclient-dev` from `apt` and then re-compiled 0.7.7 with that and it works fine now.

So it wasn't that 0.5.10 worked because it was older, it worked because it was installed from `apt` (and thus not compiled with the wrong `libmpdclient` library). I noticed this when `ncmpc` installed from `apt` worked fine but a version I compiled myself didn't.
